src/main/native/windows/processes-jni.cc

```diff
@@ -548,6 +548,22 @@ class NativeProcess {
       }
     } else if (process_ != INVALID_HANDLE_VALUE) {
       if (!TerminateProcess(process_, exit_code)) {
+        std::wstring our_error = bazel::windows::MakeErrorMessage(
+            WSTR(__FILE__), __LINE__, L"NativeProcess::Terminate",
+            ToString(pid_), GetLastError());
+
+        // If the process exited, despite TerminateProcess having failed, we're
+        // still happy and just ignore the error. It might have been a race
+        // where the process exited by itself just before we tried to kill it.
+        if (GetExitCode() == STILL_ACTIVE) {
```

Comment:
If the process exited, how can `GetExitCode()` return STILL_ACTIVE?

Reply:
Reading this again, I realize that my comment wasn't clear about this. I added some more text and now it should make sense :)

Basically, if TerminateProcess fails and GetExitCode gives STILL_ACTIVE then it means we really hit an error (and not just the race condition where we tried to terminate a process that had already exited).